content/user/running/resend_order_email.md

```diff
@@ -13,7 +13,11 @@ This can happen for a variety of reasons:
 - they need an invoice for tax or record keeping purposes. 
 - your cart has an [email deliverability issue](/user/email/emails_not_received/). 
 
-Here are a few ways you can address this need. 
+Here are a few ways you can get them the information they need. (Of course if you have a deliverability issue, you will need to handle that separately.) 
+
+## Using the Email Archive Manager 
+
+If you had [email archiving](/user/admin_pages/configuration/configuration_emailoptions#email_archiving_active) turned on at the time the order was placed, you can use the [Email Archive Manager](/user/email/email_archive_manager/) to resend the email.  (Note: You must be running Email Archive Manager version 1.9 or higher if your cart uses HTML email, and only the text copy of the email is re-sent.) 
```

Comment:
I'm inclined to put mentions of plugins at the end of the list, not the top. Emphasize how to use the software as-is first, with mention of addons later.

And the note about "use a certain version" should be on that plugin's download page.
(And in this case, since somehow we've got a docs page for this plugin, putting the version 1.9 message should be there, not here)

Reply:
Will do.